tests/integration/base_object_test.go

```diff
@@ -56,13 +56,13 @@ func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite
 	logger.Infof("Step 1 : Create Object Store User")
 	cosuErr := helper.ObjectUserClient.Create(namespace, userid, userdisplayname, storeName)
 	require.Nil(s.T(), cosuErr)
-	logger.Infof("Waiting 5 seconds to ensure user was created")
-	time.Sleep(5 * time.Second)
-	require.True(s.T(), helper.ObjectUserClient.UserSecretExists(namespace, userid, storeName), "make sure user secret was created")
+	logger.Infof("Waiting 20 seconds to ensure user was created")
+	time.Sleep(20 * time.Second)
```

Comment:
Usually it is very fast but I had a couple failures on the create which could not be explained. I would like to change this to a check look with a sleep(5).

Reply:
I changed it to a sleep(10) since a sleep(5) resulted in a failure during the first local test. I updated the secret checks to be a loop with a iteration limit, early exit and a sleep(5).